apps/source_academy/priv/repo/migrations/20170626081409_create_users.exs

```diff
@@ -0,0 +1,15 @@
+defmodule SourceAcademy.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :name, :string
+      add :email, :string
+      add :is_admin, :boolean, default: false
```

Comment:
Just to make sure we don't screw up and set it to value other than 'staff', 'student', or 'admin'. Can be updated in next PR.